### DISCUSSION TITLE
Minimum CD Meetup

### DISCUSSION THREAD

**Discussion by Jerreck** [Minimum-CD/cd-manifesto#110] [Q&A] [answered]
We met a week or so ago and figured out a good way for us to deliberate changes to the manifesto as describe in our [contribution guidelines](https://minimumcd.org/CONTRIBUTING.md).  Now I think it's time to plan our first session for actually deliberating.  

The trick is that we have people in time zones across the world, and there's not really any one hour of the day where someone wouldn't be staying up really late or waking up really early.  I think picking a Friday would probably work best since that ought to make it a little bit easier for people in that situation.  I'm going to go ahead and suggest we shoot for October 29th to give us time to let folks know and work out schedules.  

Topic for discussion then would be, what time on October 29th should we meet?  [Let's use UTC 0 as our frame of reference](https://time.is/UTC).

**Comment by bdfinst** [comment, marked as answer]
I'm good with as early as 13:00 UTC.

**Reply by ferrix** [reply]
I'm good until 14 UTC, so Bryan's suggestion would work. Fridays at EOB are for friends, fools and family, so not the best time for Western hemisphere times.

**Reply by Jerreck** [reply]
you think we should pick a different day?  what would work better?

**Reply by ferrix** [reply]
So, 13-14 UTC is possible for me on that day and there is a general rule that Friday evenings are generally offline.

**Reply by Jerreck** [reply]
scheduled for tomorrow at 13:00 UTC.  calendar invite below:

https://calendar.google.com/event?action=TEMPLATE&tmeid=MW5yMnI2ZDVkZWowanNubDMxc21qaDJnMDggbmxhZXZranVqZGZyOGxsbWlmczY2bDc4aG9AZw&tmsrc=nlaevkjujdfr8llmifs66l78ho%40group.calendar.google.com

**Reply by bdfinst** [reply]
I can't see this.

**Reply by Jerreck** [reply]
my bad - link updated

**Comment by patrickkelso** [comment]
The only times I'm definitely asleep are 1100 to 1730 UTC. I am ambivalent about what day.
